test/public/runs/runsPerDataPass.overview.test.js

```diff
@@ -178,16 +179,14 @@ module.exports = () => {
             el.value = '1111';
             el.dispatchEvent(new Event('input'));
         });
-        await waitForTimeout(100);
-        expect(Boolean(await page.$(`${amountSelectorId} input:invalid`))).to.be.true;
+        await page.waitForSelector(`${amountSelectorId} input:invalid`);
     });
 
     it('notifies if table loading returned an error', async () => {
-        await reloadPage(page);
-        await waitForTimeout(100);
+        await waitForNavigation(page, () => reloadPage(page));
```

Comment:
No need for waitForNavigation, reloadPage already do that for you

Reply:
Done, fixed everywhere